### PR TITLE
Conditional import

### PR DESCRIPTION
**Pull request by victorg1991**
This will allow import generated files to an app and still compile without change anything

### CODE REVIEW

File: Liferay-iOS-SDK.podspec [hunk 1]
```diff
@@ -25,4 +25,6 @@ Pod::Spec.new do |s|
 							}
 	s.requires_arc			= true
 	s.dependency			"AFNetworking", "= 2.6.3"
+	s.pod_target_xcconfig = { 'GCC_PREPROCESSOR_DEFINITIONS' => 'LIFERAY_MOBILE_SDK_MODULE=1' }
```
Comment (jmnavarro) [inline Liferay-iOS-SDK.podspec]: can you rename `LIFERAY_MOBILE_SDK_MODULE` by `LIFERAY_MOBILE_SDK_FRAMEWORK`?

Comment (victorg1991) [reply Liferay-iOS-SDK.podspec]: Done! :D

**Comment by nhpatt** [comment]
👍

**Comment by nhpatt** [comment]
tried it and works :)

**Comment by jmnavarro** [comment]
@allanragec can you peer-review it, please? Ask Victor any questions you have. Thanks!

**Comment by allanragec** [comment]
Ok @jmnavarro!

**Comment by allanragec** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Pull request merged**